src/coreclr/gc/env/gcenv.ee.h

```diff
@@ -96,6 +96,8 @@ class GCToEEInterface
     static void DiagAddNewRegion(int generation, uint8_t* rangeStart, uint8_t* rangeEnd, uint8_t* rangeEndReserved);
 
     static void LogErrorToHost(const char *message);
+
+    static uint64_t GetThreadIdForLogging(Thread* thread);
```

Comment:
Seems like the purpose of getting the thread id is more than just for logging - maybe we should change the name to just `GetThreadId`?

Reply:
It is logging, it is just that we are logging to a circular buffer instead of logging to a file.